docs/source/models/concept_models/vivarium_mncnh_portfolio/antenatal_care_module/module_document.rst

```diff
@@ -52,7 +52,46 @@ and :ref:`our facility choice module documentation <2024_vivarium_mncnh_portfoli
 2.0 Module Input and Output Data
 ++++++++++++++++++++++++++++++++
 
-2.1 Module Description 
+2.1 Module Inputs
+-----------------
+
+.. list-table:: Module required inputs
+  :header-rows: 1
+
+  * - Input
+    - Source 
+    - Definition
+    - Application
+    - Note
```

Comment:
If none of the rows have a Note, then we could just delete the column to make the table look better.

Reply:
Oh, but I added a note in the ANC propensity row, so never mind.